ArdusatSDK.cpp

```diff
@@ -90,7 +105,7 @@ const char * unit_to_str(uint8_t unit)
  *
  * @param sensorName name of sensor that failed.
  */
-void _beginError(prog_char *sensorName)
+void _beginError(prog_char sensorName[])
```

Comment:
I agree, but this was the preferred way when I was reading up on Arduino style concerns.